packages/@lightningjs/ui-components/src/components/FocusManager/FocusManager.mdx

```diff
@@ -119,35 +119,52 @@ class FancyFocus extends FocusManager {
 
 ### Properties
 
-| name          | type                                                    | readonly | description                                                                                                                  |
-| ------------- | ------------------------------------------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| direction     | 'column'\|'row'                                         | false    | determines whether focus handling responds to left/right or up/down key                                                      |
-| selected      | <DocsLink id="lng.Component" />                         | true     | current item in focus                                                                                                        |
-| selectedIndex | number                                                  | false    | index of current item in focus                                                                                               |
-| loading       | boolean                                                 | true     | False if an item is in focus, true otherwise                                                                                 |
-| items         | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | alias for `children`                                                                                                         |
-| wrapSelected  | boolean                                                 | false    | enables wrapping behavior, so `selectNext` selects the first item if the current item is the last on the list and vice versa |
+| name         | type            | default | description                                                        |
+| ------------ | --------------- | ------- | ------------------------------------------------------------------ |
+| direction    | 'column'\|'row' | false   | Determines the key handling for focus ('left/right' or 'up/down'). |
+| wrapSelected | boolean         | false   | Enables wrapping behavior for focus navigation.                    |
 
 ### Methods
```

Comment:
`FocusManager` has a couple more public methods that looks like haven't been added yet: `appendItemsAt()`, `getTransitionXTargetValue()`,` getTransitionYTargetValue()`

Reply:
The `FocusManager.d.ts` file has descriptions for these methods which can be used in this documentation